### PR TITLE
Add unmaintained advisories

### LINKED ISSUE #2078: get-size is unmaintained
https://crates.io/crates/get-size last updated over 1 year ago and is using the yanked crate `proc-macro-error` which was raised in #2011 

No activity at https://github.com/DKerp either.

This also includes the `get-size-derive` crate.

### PR DESCRIPTION
**Pull request by bircni**
Closes #2078

### CODE REVIEW

File: crates/get-size-derive/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# get-size-derive is unmaintained
+
+get-size-derive's maintainer seems to be unreachable, with no commits and releases pushed for 1 year and no activity on the GitHub repo.
+
+get-size-derive also depends on `attribute-derive ^0.6` a version of the crate which uses the yanked crate `proc-macro-error`.
+
+## Possible Alternative(s)
+
+- [proc-macro-error2](https://crates.io/crates/get-size-derive2)
```
Comment (GnomedDev) [inline crates/get-size-derive/RUSTSEC-0000-0000.md:21]: You forgot to change the text to `get-size-derive2`.

File: crates/get-size/RUSTSEC-0000-0000.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# get-size is unmaintained
+
+get-size's maintainer seems to be unreachable, with no commits and releases pushed for 1 year and no activity on the GitHub repo.
+
+get-size also depends on `get-size-derive 0.1.3`, which uses `attribute-derive ^0.6` a version of the crate which uses the yanked crate `proc-macro-error`.
+
+## Possible Alternative(s)
+
+- [proc-macro-error2](https://crates.io/crates/get-size2)
```
Comment (GnomedDev) [inline crates/get-size/RUSTSEC-0000-0000.md:21]: Ditto

**Review by GnomedDev — CHANGES_REQUESTED** [review]
If you could also add my `typesize` library to the alternatives list, that would be great.

File: crates/get-size-derive/RUSTSEC-0000-0000.md (hunk 1 above)
Comment (bircni) [reply crates/get-size-derive/RUSTSEC-0000-0000.md:21]: oops maybe it was a bit too late xD

**Review by GnomedDev — APPROVED** [review]
Advisories look good to me, although I think this will take a bit to wait to give them a chance to respond to the issue.

**Review by kornelski — APPROVED** [review]

**Comment by bircni** [comment]
@amousset is this ok to merge?

**Review by kornelski — APPROVED** [review]

**Comment by bircni** [comment]
90 days are over and I still got no response - I think now we can merge

**Pull request merged**